app/utils/translations/poeditor/group.ts

```diff
@@ -0,0 +1,67 @@
+export {};
```

Comment:
probably not needed. i get `SyntaxError: Unexpected token 'export'`

Reply:
Oh, I wrote this out of habit, to make TS consider the file as a module and therefore avoid some unexpected behavior from the compiler; not needed here.